akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonProxy.scala

```diff
@@ -63,6 +63,7 @@ object ClusterSingletonProxySettings {
 /**
  * @param singletonName The actor name of the singleton actor that is started by the [[ClusterSingletonManager]].
  * @param role The role of the cluster nodes where the singleton can be deployed. If None, then any node will do.
+ * @param team The team of the cluster nodes where the singleton is running. If None then the same team as current node.
```

Comment:
Indeed the current idea is a "global singleton" will be tied to a specific team: moving it between teams would only be useful in the case that a whole team (e.g. datacenter/availabilityzone) is lost. We're not currently planning any automatic "downing"/"status" tracking of whole teams, as this is such a catastrophic failure and it'd be hard to detect accurately. At this point it seems to make sense to require manual intervention.